ocean.js

```diff
@@ -0,0 +1,251 @@
+import * as main from "./main.js";
+//  MAPBOX
+mapboxgl.accessToken =
+  "pk.eyJ1Ijoib3dudG9vbWFueW1vbnN0ZXIiLCJhIjoiY2tvNzMwMGpuMjk4ZDJvbXZqaHBqanlpbSJ9.DHukChuKakMnzj-mieZ1Og";
```

Comment:
I highly recommend to not putting your API key in here. Everyone can see and what is more problematic, use it. At your expense! Please use a `.env` file and don't put any secrets into the codebase or into git.

Reply:
Okay, wouldnt do it if tit would be another scenario! ;) Iwill have a look at .env  Thx